components/Navbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import Link from "next/link"
+import styles from '../styles/Nav.module.css'
+
+const Navbar = () => {
+  return (
+    <ul className={styles.ul}>
+      <li className={styles.li}>
+        <Link href="/">
+          <a>Home</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/register">
+          <a>Register</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/login">
+          <a>Login</a>
+        </Link>
+      </li>
+    </ul>
+  )
+}
+
+export default Navbar

```

Comment:
Please make use of regular `export` for components, utilities etc. Make use of `export default` only in pages components - because that is what next.js specifies. 

I would recommend limiting the use of `export default` due to it not being that friendly to autocomplete imports. And devs easily abusing the naming.

e.g.

```
import HindiAkoNavBar from 'components/Navbar'

vs. 

import { Navbar } from 'components/Navbar'
```